src/script/vco2_main/factions/wh2_dlc15_hef_imrik/campaigns/main_warhammer/missions.lua

```diff
@@ -1,186 +1,160 @@
+                    province wh2_main_cothique;
+                    province wh2_main_avelorn;
+                    province wh2_main_chrace;
+                    province wh2_main_nagarythe;
+                    province wh2_main_ellyrion;
+                    province wh2_main_tiranoc;
+                    province wh2_main_caledor;
+                    province wh2_main_eagle_gate;
+                    province wh2_main_griffon_gate;
+                    province wh2_main_unicorn_gate;
+                    province wh2_main_phoenix_gate;
+                    province wh2_main_gnoblar_country;
+                    province wh2_main_dragon_isles;
+                    province wh2_main_the_wolf_lands;
+                    province wh2_main_the_plain_of_bones;
+                }
+                objective
+                {
+                    type DESTROY_FACTION;
+                    faction wh2_main_skv_clan_eshin;
+                    faction wh2_main_skv_clan_mors;
 
-					faction wh_main_grn_greenskins;
+                    faction wh_main_grn_greenskins;
 
-					faction wh2_main_def_hag_graef;
-					confederation_valid;
-				}
-				objective
-				{
-					type CONSTRUCT_N_BUILDINGS_FROM;
-					total 8;
-					building_level wh2_main_hef_mages_3;
-					building_level wh2_main_hef_worship_3;
-					building_level wh2_main_hef_dragons_2;
-					building_level wh2_main_hef_smith_2;
-					building_level wh2_dlc10_hef_handmaidens;
-					building_level wh2_main_hef_defence_major_3;
-					building_level wh2_main_hef_order_5;
-					building_level wh2_main_hef_port_3;
-					faction wh2_main_hef_caledor;
-				}
-				objective
+                    faction wh2_main_def_hag_graef;
+                    confederation_valid;
+                }
+                objective
                 {
                     type CONSTRUCT_N_BUILDINGS_INCLUDING;
                     faction wh2_dlc15_hef_imrik;
-                    total 2;
+                    total 3;
                     building_level wh2_dlc15_special_graves_of_the_dragons_2_hef;
+                    building_level wh2_main_hef_dragons_2;
                     building_level wh2_main_special_hall_of_dragons;
                 }
-				objective
-				{
-					type OWN_N_UNITS;
-					total 15;
-					unit wh2_main_hef_mon_sun_dragon;
-					unit wh2_dlc10_hef_inf_the_silverpelts_ror_0;
-					unit wh2_main_hef_inf_phoenix_guard;
-					unit wh2_main_hef_cav_dragon_princes;
-					unit wh2_main_hef_mon_star_dragon;
-					unit wh2_dlc10_hef_cav_the_fireborn_ror_0;
-					unit wh2_dlc10_hef_inf_keepers_of_the_flame_ror_0;
-					unit wh2_dlc10_hef_inf_sisters_of_avelorn_0;
-					unit wh2_dlc10_hef_inf_everqueens_court_guards_ror_0;
-					unit wh2_main_hef_inf_white_lions_of_chrace_0;
-				}
-				objective
-				{
-					type CONTROL_N_PROVINCES_INCLUDING;
-					total 18;
-				}
+                objective
+                {
+                    type OWN_N_UNITS;
+                    total 15;
```

Comment:
I'll change the Dragon unit requirements from 5/15 to 4/8. We can discuss it in the PR thread if you want to.